tests/geojson_tiler_test_data/geojson_1/oneBlock.geojson

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What about the ownership (and source) of all the geojson files ?
Would it hurt to place a [json comment](https://codersdiaries.com/blog/comments-in-json) to place such copyright information within the file themselves ?